src/main/webapp/app/shared-ui/resizeable-container/resizeable-container.component.scss

```
@@ -85,6 +85,8 @@
         justify-content: center;
         min-width: 15px;
         z-index: 10;
+        // Show a resize affordance on the drag handle (interact.js used to set this automatically).
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)